src/test/scala/nl/vroste/zio/kinesis/client/LocalStackLayers.scala

```diff
@@ -0,0 +1,85 @@
+package nl.vroste.zio.kinesis.client
+
+import java.net.URI
+
+import software.amazon.awssdk.auth.credentials.{
+  AwsBasicCredentials,
+  AwsCredentialsProvider,
+  StaticCredentialsProvider
+}
+import software.amazon.awssdk.core.SdkSystemSetting
+import software.amazon.awssdk.http.SdkHttpConfigurationOption
+import software.amazon.awssdk.http.async.SdkAsyncHttpClient
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.cloudwatch.CloudWatchAsyncClient
+import software.amazon.awssdk.services.dynamodb.DynamoDbAsyncClient
+import software.amazon.awssdk.services.kinesis.KinesisAsyncClient
+import software.amazon.awssdk.utils.AttributeMap
+import zio.{ Has, ZIO, ZLayer, ZManaged }
+
+object LocalStackLayers {
```

Comment:
What do you think of `LocalStackServices` instead of `..Layers`?

Reply:
done